timescaledb/tutorials/nfl-fantasy-league.md

```diff
@@ -0,0 +1,188 @@
+# Win your NFL fantasy league with TimescaleDB
+
+This tutorial is a step-by-step guide on how to ingest and analyze american football data with TimescaleDB.
+
+The dataset that we're using is provided be the National Football League (NFL) and contains data about 
```

Comment:
"The dataset we're using is provided **BY** the..."

Reply:
https://github.com/timescale/docs/pull/187/commits/6a12d2bc2512699e3004e5cd0a3da6bf3ce7a39f